Changelog.rst

```diff
@@ -100,8 +122,9 @@ variable, set in ``manifests/site.pp``, that controls the list of classes to be
 included. The goal of this change is to ease users with existing infrastructures
 into using full-bore SIMP.
 
-Essentially, ``simp_classes.yaml`` has been replaced by ``scenarios/simp.yaml``
-and ``simp_def.yaml`` has been replaced by ``simp_config_settings.yaml``.
+Essentially, ``simp_classes.yaml`` has been replaced by class inclusions under
+the ``simp::scenario`` namespace and ``simp_def.yaml`` has been replaced by
+``simp_config_settings.yaml``.
```

Comment:
We shouldn't advise users to modify `simp_config_settings.yaml`. simp config will overwrite their changes here. They can modify the settings in `simp_config_settings.yaml` in `default.yaml`.